.github/workflows/build-depends.yml

```
@@ -119,20 +130,20 @@ jobs:
       - name: Restore cached depends
-        uses: actions/cache/restore@v4
+        uses: actions/cache/restore@v5
         with:
           path: depends/built/${{ needs.check-cache.outputs.host }}
           key: ${{ needs.check-cache.outputs.cache-key }}
           restore-keys: |
-            depends-${{ hashFiles('contrib/containers/ci/ci.Dockerfile', 'contrib/containers/ci/ci-slim.Dockerfile') }}-${{ inputs.build-target }}-
+            ${{ needs.check-cache.outputs.cache-key-prefix }}-
 
       - name: Build depends
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:2:54: Quote this to prevent word splitting

(shellcheck)